ci/unit/test_post_always.sh

```diff
@@ -13,17 +13,32 @@ else
   exit 1
 fi
 
-rm -rf run_test.sh vm_test
 DAOS_BASE="${SL_PREFIX%/install*}"
 NODE="${NODELIST%%,*}"
-
 mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 
-# shellcheck disable=SC2029
-ssh "$SSH_KEY_ARGS" jenkins@"$NODE" \
-  "DAOS_BASE=$DAOS_BASE             \
-  $(cat "$mydir/test_post_always_node.sh")"
+if [ $# -ge 1 ] && [ -n "$1" ]; then
+    if [ "$1" == "memcheck" ]; then
+        echo "run test post always with memcheck"
+        rm -rf run_test_memcheck.sh
+        WITH_VALGRIND=memcheck
+        # shellcheck disable=SC2029
+        ssh "$SSH_KEY_ARGS" jenkins@"$NODE" \
+          "DAOS_BASE=$DAOS_BASE             \
+          WITH_VALGRIND=$WITH_VALGRIND      \
+          $(cat "$mydir/test_post_always_node.sh")"
+    fi
+else
+    echo "run test post always"
+    rm -rf run_test.sh vm_test
+    WITH_VALGRIND=disabled
+    # shellcheck disable=SC2029
+    ssh "$SSH_KEY_ARGS" jenkins@"$NODE" \
+      "DAOS_BASE=$DAOS_BASE             \
+      WITH_VALGRIND=$WITH_VALGRIND      \
+      $(cat "$mydir/test_post_always_node.sh")"
```

Comment:
Again, too much repetitive code.  There should only be one `ssh` command here.

Reply:
I'll fix on the next patch